lms/static/scripts/frontend_apps/components/test/FilePickerApp-test.js

```diff
@@ -506,6 +508,91 @@ describe('FilePickerApp', () => {
     assert.isFalse(wrapper.exists('ErrorModal'));
   });
 
+  context('when promptForGradable is enabled', () => {
+    const deepLinkingAPIPath = '/lti/1.3/deep_linking/form_fields';
+    const deepLinkingAPIData = { some: 'data' };
+    let fakeAPICall;
+    let fakeFormFields;
+
+    beforeEach(() => {
+      fakeConfig.filePicker.deepLinkingAPI = {
+        path: deepLinkingAPIPath,
+        data: deepLinkingAPIData,
+      };
+      fakeConfig.filePicker.promptForGradable = true;
+
+      fakeAPICall = sinon.stub();
+      fakeFormFields = { JWT: 'JWT VALUE' };
+
+      fakeAPICall
+        .withArgs(sinon.match({ path: deepLinkingAPIPath }))
+        .resolves(fakeFormFields);
+
+      $imports.$mock({
+        '../utils/api': { apiCall: fakeAPICall },
+      });
+    });
+
+    it('displays "max points" input on details screen', () => {
+      const wrapper = renderFilePicker();
+      selectContent(wrapper, 'https://example.com');
+      const gradableCheckbox = wrapper.find(
+        'input[data-testid="gradable-max-input"]',
+      );
+      assert.isTrue(gradableCheckbox.exists());
+    });
```

Comment:
Maybe it would be good to verify the input is there when `promptForGradable: true`, and not there when `promptForGradable: false`.

We usually do something on these lines:

 ```suggestion
    [true, false].forEach(promptForGradable => {
      it('displays "max points" input on details screen when promptForGradable is enabled', () => {
        fakeConfig.filePicker.promptForGradable = promptForGradable;
      
        const wrapper = renderFilePicker();
        selectContent(wrapper, 'https://example.com');
        const gradableCheckbox = wrapper.find(
          'input[data-testid="gradable-max-input"]',
        );
        
        assert.equal(gradableCheckbox.exists(), promptForGradable);
      });
    });
```